packages/web-react/src/components/ButtonLink/__tests__/useButtonLinkAriaProps.test.ts

```diff
@@ -0,0 +1,88 @@
+import { renderHook } from '@testing-library/react';
+import { UseButtonLinkAriaProps, useButtonLinkAriaProps } from '../useButtonLinkAriaProps';
+
+describe('useButtonAriaProps', () => {
+  it('should return aria props for anchor tag', () => {
+    const props = {
+      elementType: 'a',
+      href: '/',
+      isDisabled: false,
+      target: '_blank',
+      rel: 'noopener',
+    } as UseButtonLinkAriaProps;
+    const { result } = renderHook(() => useButtonLinkAriaProps(props));
+
+    expect(result.current.buttonLinkProps).toEqual({
+      'aria-label': undefined,
+      disabled: undefined,
+      href: '/',
+      onClick: expect.any(Function),
+      rel: 'noopener',
+      role: 'button',
+      target: '_blank',
+    });
+  });
+
+  it('should return aria props for disabled anchor tag', () => {
+    const props = {
+      elementType: 'a',
+      href: '/',
+      isDisabled: true,
+      target: '_blank',
+      rel: 'noopener',
+    } as UseButtonLinkAriaProps;
+    const { result } = renderHook(() => useButtonLinkAriaProps(props));
+
+    expect(result.current.buttonLinkProps).toEqual({
+      'aria-label': undefined,
+      disabled: true,
+      href: undefined,
+      onClick: expect.any(Function),
+      rel: 'noopener',
+      role: 'button',
+      target: '_blank',
+    });
+  });
+
+  it('should return aria props for button tag', () => {
+    const props = {
+      elementType: 'button',
+      href: '/',
```

Comment:
**question (if-minor):** 🤔 `href` should not be passed to the button element, IMHO. Maybe there is another problem with the condition.

Reply:
The same for the `target` and `rel`. I think.